core/domain/exp_domain.py

```diff
@@ -1961,7 +1961,8 @@ def _convert_states_v47_dict_to_v48_dict(cls, states_dict):
             if interaction_customisation_args:
                 state_domain.State.convert_html_fields_in_state(
                     state_dict,
-                    html_validation_service.fix_incorrectly_encoded_chars)
+                    html_validation_service.fix_incorrectly_encoded_chars,
+                    state_schema_version=48)
```

Comment:
I'm not sure whether this is a proper fix i.e, `InteractionCustomizationArg` is a class method and expected to change whenever the state-schema is changing and in such migrations the old conversion function will start breaking, right?

Have we verified whether the migration works after this change i.e, there might be cases where we have changed the schema of another part of the state?

Also, do you have any suggestions on how we can solve it in general?